projects/wave-core-new/src/lib/tab-panel/tab-panel.component.ts

```diff
@@ -0,0 +1,154 @@
+import {CdkPortalOutlet, ComponentPortal} from '@angular/cdk/portal';
+import {
+    Component,
+    OnInit,
+    ChangeDetectionStrategy,
+    HostBinding,
+    Input,
+    OnChanges,
+    SimpleChanges,
+    OnDestroy,
+    ViewChild,
+    ComponentFactoryResolver,
+    Injector,
+    AfterViewInit,
+    ChangeDetectorRef,
+} from '@angular/core';
+import {Subscription} from 'rxjs';
+import {Config} from '../config.service';
+import {LayoutService} from '../layout.service';
+import {TabContent, TabPanelService} from './tab-panel.service';
+
+@Component({
+    selector: 'wave-tab-panel',
+    templateUrl: './tab-panel.component.html',
+    styleUrls: ['./tab-panel.component.scss'],
+    changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class TabPanelComponent implements OnInit, OnChanges, OnDestroy, AfterViewInit {
```

Comment:
maybe just call it TabComponent? A Tab panel is only the stuff corresponding to a tab https://www.w3.org/wiki/TabPanel

Reply:
d280776